aws/resource_aws_kinesis_stream.go

```diff
@@ -12,6 +13,12 @@ import (
 	"github.com/hashicorp/terraform/helper/schema"
 )
 
+// Number of attempts to create the Kinesis streams
+const KINESIS_MAX_ATTEMPTS = 10
+
+// How long to sleep if a limit-exceeded event happens
+const KINESIS_LIMIT_EXCEEDED_SLEEP = 10 * time.Second
+
```

Comment:
Probably a nitpick, but these constants are only ever going to be used in this file in a single place, so we may as well just hard code those values and avoid the extra 5 LOC.